mappings/net/minecraft/structure/StructurePiece.mapping

```diff
@@ -172,7 +172,7 @@ CLASS net/minecraft/class_3443 net/minecraft/structure/StructurePiece
 		ARG 6 y
 		ARG 7 z
 		ARG 8 state
-	METHOD method_14946 getTag ()Lnet/minecraft/class_2487;
+	METHOD method_14946 getNbt ()Lnet/minecraft/class_2487;
```

Comment:
```suggestion
	METHOD method_14946 toNbt ()Lnet/minecraft/class_2487;
```
(this does `writeNbt` with a new compound tag and adds some extra properties to it, including the ID and bounding box)